content/04.results.md

```diff
@@ -98,21 +102,98 @@ This is the revision of the first paragraph of the introduction of CCC:
+There is no mention of the CRISPR screen and the gene symbols identified to be associated with the regulation of lipids, which are the key elements in the original text.
+Instead, the model seemed to have focused more on the title and keywords of the manuscript (Table @tbl:manuscripts) that are part of every prompt (Figure @fig:ai_revision).
+It included the idea of a "gene co-expression" analysis (a keyword) to identify "therapeutic targets" (another keyword), and replaced the mention of "sets of genes" in the original text with "clusters of genes" in the revision (closer to the "clustering" keyword).
+Although this is clearly a bad revision, the output of the model also suggests that the original paragraph is maybe too short or disconnected from the rest, and that it should probably be merged with the next one (which describes follow-up and related experiments).
+
+
 ### Discussion
 
+All of the paragraphs in the Discussion sections of CCC and PhenoPLIER had a high quality revision.
+The model kept the correct format when necessary (e.g., using italics for gene symbols), maintained most of the citations, and improved the readability of the text in general.
+Some paragraphs had minor mistakes that are easy to fix by a human author.
+
+Below we show a paragraph of CCC where we discuss how not-only-linear correlation coefficients could potentially impact genetic studies of complex traits:
+
+![
+](images/diffs/discussion/ccc-paragraph-01.svg "Diffs - CCC discussion paragraph 01"){width="100%"}
+
+Although some minor changes could be added, we believe the revised text reads better than the original.
+It is also interesting how the model understood the format of citations and built more complex structures from it.
+For instance, the two articles referenced in lines 2 and 3 in the original text were correctly merged into a single citation block and separated with ";" in line 2 of the revised text.
+
+
 ### Methods
 
+Prompts for the Methods section were the most challenging to design, especially when it comes to the definition of equations.
+The prompt for Methods (Figure @fig:ai_revision) is more focused in keeping the technical details, which was especially important for PhenoPLIER, whose Methods section contains paragraphs with several mathematical expressions.
+
+We show below the revision of a paragraph in PhenoPLIER that contains two numbered equations:
+
+![
+](images/diffs/methods/phenoplier-paragraph-01.svg "Diffs - PhenoPLIER methods paragraph 01"){width="100%"}
+
+The revised text contains very few changes: all the equations, citations and most of the original text was preserved.
+However, it is interesting how the model identified a mistake in the original text (line 8) and fixed it in the revision (line 7).
+Indeed, the equation with the univariate model used by PrediXcan (lines 4 to 6 in the original) includes the *true* effect size $\gamma_l$ (`\gamma_l`) instead of the *estimated* one $\hat{\gamma}_l$ (`\hat{\gamma}_l`).
```

Comment:
Am I reading this correctly that this change from GPT-3 is correct? Below, it looks to me like it's doing the opposite unless I'm misunderstanding.

Reply:
Yes, the GPT-3 model is doing the right thing: it corrected a wrong reference to a mathematical symbol.

Reply:
I tried to improve the clarity here by splitting the paragraph.